lib/default-config.js

```diff
@@ -1050,7 +1050,7 @@ module.exports = function (config) {
       choices = choices.slice(0);
 
       // Array of choice arrays should be flattened.
-      choices = _.flatten(choices);
+      choices = _.compact(_.flatten(choices));
```

Comment:
Just to keep things interesting, formatic doesn't use lodash or underscore but rather its own undash lib: https://github.com/zapier/formatic/blob/28cb22cc9d970c8f9da5efe2e725799ca8e07b78/lib/default-config.js#L14. :-)

So you'll need to add `_.compact` to that.

This was done to keep dependencies very small. Probably should have not called it `_` though and just called it 'utils' or something.